web/src/styles/office-tour.css

```
@@ -0,0 +1,255 @@
+  background: var(--accent);
+}
+
+/* ─── Synchronous-fallback live region (visually hidden, SR-available) ─── */
+.office-tour-transition-fallback {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  margin: -1px;
+  padding: 0;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
```

Stylelint (17.12.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)